### PR TITLE
fix bceloss weight

### PR DESCRIPTION
**Pull request by ceci3**
fix weight in bceloss 
image

### CODE REVIEW

File: python/paddle/nn/layer/loss.py [hunk 1]
```diff
@@ -224,40 +224,55 @@ def forward(self, input, label):
 
 class BCELoss(fluid.dygraph.Layer):
     """
-    This op accepts input predictions and target label and returns binary 
-    cross entropy error.
-    For predictions label, and target label, the loss is calculated as follows.
+    This op measures the binary cross entropy between input predictions and target label. 
+    The BCELoss layer calculates the binary cross entropy loss as follows.
```
Comment (wanghaoshuang) [inline python/paddle/nn/layer/loss.py]: 上句说op，这里又是layer，最好统一下。
label -> labels

Comment (wanghaoshuang) [reply python/paddle/nn/layer/loss.py]: cross entropy -》cross-entropy

Comment (wanghaoshuang) [reply python/paddle/nn/layer/loss.py]: As follows means what comes next, or the things listed here, and is usually followed by a colon (:).

File: python/paddle/nn/layer/loss.py [hunk 2]
```diff
@@ -224,40 +224,55 @@ def forward(self, input, label):
...
+
     .. math::
         Out = MEAN(Out)
     If :attr:`reduction` set to ``'sum'``, the reduced sum loss is:
+
     .. math::
         Out = SUM(Out)
+
     Parameters:
         input (Variable): Input tensor, the data type is float32,
```
Comment (wanghaoshuang) [inline python/paddle/nn/layer/loss.py]: 动态图的参数列表都是这么列么？forward的参数也写到构造函数这儿？

File: python/paddle/nn/layer/loss.py [hunk 3]
```diff
@@ -224,40 +224,55 @@ def forward(self, input, label):
...
-            float64. Input must in (0, 1).
+            float64. Input must be in [0, 1].
         label (Variable): Label tensor, has the same shape with input,
```
Comment (wanghaoshuang) [inline python/paddle/nn/layer/loss.py]: 你这语法肯定不对

File: python/paddle/nn/layer/loss.py [hunk 4]
```diff
@@ -224,40 +224,55 @@ def forward(self, input, label):
...
-            float64. Input must in (0, 1).
+            float64. Input must be in [0, 1].
         label (Variable): Label tensor, has the same shape with input, 
-            the data type is float32, float64.
-        weight (Variable, optional): Weight tensor, a manual rescaling weight given
-            to each class. It has the same dimensions as class number and the data type
-            is float32, float64, int32, int64. Default is ``'None'``.
+            the data type is float32, float64. Note that the label should be numbers
+            between 0 and 1.
+        weight (Variable, optional): Weight tensor, a manual rescaling weight given
```
Comment (wanghaoshuang) [inline python/paddle/nn/layer/loss.py]: `Weight tensor` 没必要加这两个单词。

File: python/paddle/nn/layer/loss.py [hunk 5]
```diff
@@ -224,40 +224,55 @@ def forward(self, input, label):
...
+            nbatch and the data type is float32, float64, int32, int64. Default is ``'None'``.
         reduction (str, optional): Indicate how to average the loss by batch_size, 
             the candicates are ``'none'`` | ``'mean'`` | ``'sum'``.
+            If :attr:`reduction` is ``'none'``, the unreduced loss is returned;
             If :attr:`reduction` is ``'mean'``, the reduced mean loss is returned; 
+            If :attr:`reduction` is ``'sum'``, the summed loss is returned.
             Default is ``'mean'``.
+
     Returns:
         The tensor variable storing the bce_loss of input and label.
```
Comment (wanghaoshuang) [inline python/paddle/nn/layer/loss.py]: tensor variable， 这种写法合适么？pytorch文档里都怎么写的？

**Comment by wanghaoshuang** [comment]
请斟酌下文档的语法

**Review by wanghaoshuang — APPROVED** [review]
